nextcloud/tests/compose/nextcloud-standalone

```diff
@@ -2,7 +2,7 @@ version: '3.5'
 
 services:
   nextcloud-standalone:
-    image: "nextcloud:latest"
+    image: "nextcloud:14.0.14"
```

Comment:
That's an image from Sept 2019. I guess that's fine for now to fix the test, but it should be fixed with latest image.

Reply:
Usually pinning to `latest` image is a bad idea because then tests break unexpectedly. but you can open a PR later to change it to a newer image